### DISCUSSION TITLE
Copilot not working in VSCode

### DISCUSSION THREAD

**Discussion by ghost** [community/community#14793] [Copilot Conversations]
Getting an error saying github copilot could not connect to server, Certificate has expired.
The full error is given below:
[ERROR] [default] [2022-04-12T15:53:41.355Z] GitHub Copilot could not connect to server. Extension activation failed: "certificate has expired"
![image](https://user-images.githubusercontent.com/103211216/163005493-7c19c43a-6e51-439f-b1c1-340b5b626efb.png)

**Comment by davecheney** [comment]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Comment by fourinlc** [comment]
Hava you solved it?I hava the same problem.

**Reply by davecheney** [reply]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?
